static/templates/presence_row.hbs

```diff
@@ -5,8 +5,18 @@
           href="{{href}}"
           data-user-id="{{user_id}}"
           data-name="{{name}}">
-            <span class="user-name">{{name}}</span>
-            {{> status_emoji status_emoji_info}}
+            {{#if right_sidebar_style.COMPACT}}
+                <span class="user-name">{{name}}</span>
+                {{> status_emoji status_emoji_info}}
+            {{else if right_sidebar_style.WITH_STATUS}}
+                <div>
```

Comment:
Remind me what the purpose of this outer div is? I would expect a div without any classes on it to not do much -- is the idea to reset to `display: block` for the inside of the `a` element?

Reply:
> Remind me what the purpose of this outer div is?

I was looking for ways to get the status text on a different line and this i what I settled with.

> is the idea to reset to `display: block` for the inside of the `a` element?

Yes